src/job/MomoJobBuilder.ts

```diff
@@ -24,7 +24,7 @@ export class MomoJobBuilder {
     return this;
   }
 
-  withSchedule(interval: string, firstRunAfter = 0): MomoIntervalJobBuilder {
+  withSchedule(interval: string, firstRunAfter: number | string = 0): MomoIntervalJobBuilder {
```

Comment:
We could consider making it possible to specify the interval in ms as well. But probably not in this PR.